docs/cloud-native-security/vuln-management-get-started.asciidoc

```diff
@@ -0,0 +1,54 @@
+[[vuln-management-get-started]]
+= Get started with CNVM
+
+This page explains how to set up Cloud Native Vulnerability Management (CNVM).
+
+.Requirements
+[sidebar]
+--
+* CNVM is available to all {ecloud} users. On-premise deployments require an https://www.elastic.co/pricing[Enterprise subscription].
+* {stack} and {agent} versions 8.8 or higher.
+* To view vulnerability scan findings, you need at least `read` privileges for the following indices:
+** `logs-cloud_security_posture.vulnerabilities-*`
+** `logs-cloud_security_posture.vulnerabilities_latest-*`
+--
+
+NOTE: CNVM currently only supports AWS EC2 Linux workloads.
+
+[discrete]
+[[vuln-management-setup]]
+== Set up CNVM for AWS
+
+To set up the CNVM integration for AWS, install the integration on a new {agent} policy, sign into the AWS account you want to scan, and run the https://docs.aws.amazon.com/cloudformation/index.html[CloudFormation] template.
+
+[discrete]
+[[vuln-management-setup-step-1]]
+=== Add the CNVM integration
+
+. Go to the {es-sec} Get started page, and click *Add security integrations*.
+. Search for `Cloud Native Vulnerability Management`, then click on the result.
+. Click *Add Cloud Native Vulnerability Management*.
+. Give your integration a name that matches its purpose or the region of the AWS account you want to scan for vulnerabilities, for example, `uswest2-aws-account`.
+. Click *Save and continue*. The integration will create a new {agent} policy.
+. Click *Add {agent} to your hosts*.
+
+IMPORTANT: Do not add the integration to an existing {agent} policy. It should always be added to a new policy, since it should not run on your VMs with existing workloads. For more information refer to <<vuln-management-overview-how-it-works, Vulnerability Management Overview>>.
+
+[discrete]
+[[vuln-management-setup-step-2]]
+=== Sign in to the AWS management console
```

Comment:
My thinking on making it a separate section is that you have to context switch to complete the steps in this section (need to switch tabs and sign in to AWS), so I figure it may help people keep track of where they are in the instructions when they switch back to the docs. Happy to streamline here if you prefer

Reply:
I had the same thought as @jmikell821 at first, but I get @benironside's approach too. One idea is to make the three sections one procedure, with three main steps each with their own substeps:

> 1. Add the CNVM integration
>     a. In the Elastic Security app, go to...
>     b. Search for Cloud...
>     c. Click Add Cloud... 
>     d. etc.
> 
> 2. Sign in to the AWS management console
>     a. Open a new browser
>     b. Switch to the cloud region
> 
> 3. Run the CloudFormation template
>     a. Switch back...
>     b. Click Launch...

Or, revise the headings to indicate they're all part of the same overarching procedure (the user has to do them all, in a certain order; they're not like normal sections where you can skip around more freely):

> === Step 1: Add the CNVM integration
> === Step 2: Sign in to the AWS management console
> === Step 3: Run the CloudFormation template